apps/dashboard/test/test_helper.rb

```diff
@@ -118,6 +118,13 @@ def stub_sinfo
       end
     end
 
+    # helper to stub clusters to be [ascend] becuase adding new cluster.d
+    # files may conflict with existing tests.
+    def stub_ascend
+      ascend = OodCore::Cluster.new({ id: 'ascend', job: { adapter: 'slurm' } })
+      OodAppkit.stubs(:clusters).returns(OodCore::Clusters.new([ascend]))
+    end
```

Comment:
It's easier to just have a helper stub clusters than to provide a new cluster.d file that could conflict with other tests.